README.md

```diff
@@ -1,67 +1,42 @@
-# laravel-broacast-app
-A test app for laravel broadcast using ably
+# Ably Laravel Broadcast App
 
-<p align="center"><a href="https://laravel.com" target="_blank"><img src="https://raw.githubusercontent.com/laravel/art/master/logo-lockup/5%20SVG/2%20CMYK/1%20Full%20Color/laravel-logolockup-cmyk-red.svg" width="400"></a></p>
+A demo chat application written in Vue.js for testing out the Ably Broadcaster for Laravel.
 
-<p align="center">
-<a href="https://travis-ci.org/laravel/framework"><img src="https://travis-ci.org/laravel/framework.svg" alt="Build Status"></a>
-<a href="https://packagist.org/packages/laravel/framework"><img src="https://img.shields.io/packagist/dt/laravel/framework" alt="Total Downloads"></a>
-<a href="https://packagist.org/packages/laravel/framework"><img src="https://img.shields.io/packagist/v/laravel/framework" alt="Latest Stable Version"></a>
-<a href="https://packagist.org/packages/laravel/framework"><img src="https://img.shields.io/packagist/l/laravel/framework" alt="License"></a>
-</p>
+The application features the following:
+* Laravel built-in user authentication (login, registration)
+* Public chat rooms (no client permission to send messages, only broadcast supported)
+* Private chat rooms (chat between several participants)
+* Presence channels (visible join/leave events and the number of participants)
+* Sample implementation of a typing indicator
+* Joining multiple chat rooms simultaneously
 
-## About Laravel
+## Requirements
 
-Laravel is a web application framework with expressive, elegant syntax. We believe development must be an enjoyable and creative experience to be truly fulfilling. Laravel takes the pain out of development by easing common tasks used in many web projects, such as:
+This application uses Laravel 8, which requires PHP version 7.3 or greater.
 
-- [Simple, fast routing engine](https://laravel.com/docs/routing).
-- [Powerful dependency injection container](https://laravel.com/docs/container).
-- Multiple back-ends for [session](https://laravel.com/docs/session) and [cache](https://laravel.com/docs/cache) storage.
-- Expressive, intuitive [database ORM](https://laravel.com/docs/eloquent).
-- Database agnostic [schema migrations](https://laravel.com/docs/migrations).
-- [Robust background job processing](https://laravel.com/docs/queues).
-- [Real-time event broadcasting](https://laravel.com/docs/broadcasting).
+## Setup
 
-Laravel is accessible, powerful, and provides tools required for large, robust applications.
+1. Run `composer install` and `npm install` after cloning the project
```

Comment:
1. Clone the repository
`git clone https://github.com/ably-labs/laravel-broadcast-app`
2. Install dependencies 
```
composer install
npm install
```
3. Create `.env` file, and copy contents from `.env.example`, add entry
`
ABLY_KEY=ABLY_ROOT_API_KEY
`
4. Start laravel backend server 
```
php artisan serve 
```
5. Start UI server in watch/hot-reloading mode
```
npm run watch
```
6. Access the web app via http://127.0.0.1:8000